RELEASE/scripts/autoscend/combat/auto_combat.ash

```diff
@@ -81,6 +81,16 @@ string auto_combatHandler(int round, monster enemy, string text)
 	{
 		awol_combat_helper(text);
 	}
+	if(in_pokefam())
+	{
+		if(svn_info("Ezandora-Helix-Fossil-branches-Release").revision > 0)
+		{
+		auto_log_info("Combat via Ezandora:", "green");
+		boolean ignore = cli_execute("Pocket Familiars");
+		return true;
+		}
+	}
```

Comment:
`string auto_combatHandler(int round, monster enemy, string text)`
wants a string return not a boolean return. this will not work properly.

Reply:
well, it might work actually if the ezandora script then performs the entire combat. and then when combat is over we return true into the void which does nothing?

Reply:
I can confirm through my testing that this handles the entire combat on its own and when combat ends it moves to post_adv.  I’m terrible at coding so there is probably a cleaner way to do this but it definitely works to handle all combat

Reply:
ah ok. In that case the return true just gets lost. I guess its ok for now until proper integration can be done.

Reply:
```suggestion
	if(in_pokefam())
	{
		if(svn_info("Ezandora-Helix-Fossil-branches-Release").revision > 0)
		{
		auto_log_info("Combat via Ezandora:", "green");
		boolean ignore = cli_execute("Pocket Familiars");
		return "";		//does not matter what it returns here. the cli_execute above does the entire combat
		}
	}
```